ttnn/ttnn/operations/core.py

```diff
@@ -617,7 +617,13 @@ def _to_layout_validate_input_tensors(operation_name, input_tensor, *args, **kwa
 
 
 @ttnn.register_operation(name="ttnn.to_layout", validate_input_tensors=_to_layout_validate_input_tensors)
-def to_layout(tensor, layout: ttnn.Layout, dtype: ttnn.DataType = None):
+def to_layout(
+    tensor,
+    layout: ttnn.Layout,
+    dtype: ttnn.DataType = None,
+    output_memory_config: ttnn.MemoryConfig = None,
+    use_multicore: bool = False,
```

Comment:
why is this extra function parameter needed?

Reply:
to allow you to set an output memory config and whether or not to use the multicore variant of the op.

Reply:
Please rename `output_memory_config` to `memory_config`. We don't add the prefix `output` in `ttnn` because it's obvious it's for the output and makes the name shorter.

Reply:
Do we need `use_multicore` in `ttnn`? Can we just always set it to true when calling underlying `tt_lib` operations?

Reply:
There is an issue with mutli core untilize. @tt-aho can confirm if we can turn on multi core by default in ttnn.to_layout function.